contracts/Core/AssetManager.sol

```diff
@@ -11,20 +11,19 @@ import "./ACL.sol";
 contract AssetManager is ACL, AssetStorage, Constants, StateManager, IAssetManager {
     IParameters public parameters;
 
-    event JobCreated(
+    event JobCreated(Structs.Job job, uint256 timestamp);
```

Comment:
had to modify this to struct from direct elements, as there was stack too deep error
https://app.circleci.com/pipelines/github/razor-network/contracts/562/workflows/8209fb1a-2f02-42ef-ad65-84d95dcf75b5/jobs/3367